src/lib/cap.c

```diff
@@ -200,6 +202,78 @@ void build_cap_str(cap_param *new, cap_param *old, const char *delim,
 	}
 }
 
+/** only retrun in string format for output.
+ *
+ * @param cap		capability mask.
+ * @param buf		merged capabilities in string format.
+ * @param len		max size of buf string.
+ */
+void cap_param2str(cap_param *cap, const char *delim,
+		char *buf, int len)
+{
+	unsigned int i;
+	unsigned int j = 0;
+	int r;
+	char *sp, *ep;
+	
+	unsigned int l = 0;
+	char lc[64] = "\0";
+
+	sp = buf;
+	ep = buf + len;
+	for (i = 0; i < ARRAY_SIZE(cap_names); i++) {
+		int op = 0;
+
+		if (CAP_TO_MASK(i) & cap->on)
+			op = 1;
+		else if (CAP_TO_MASK(i) & cap->off)
+			op = 2;
+		else
+			continue;
+		for (l=0; cap_names[i][l]; l++)
+			lc[l] = tolower(cap_names[i][l]);
+
+		r = snprintf(sp, ep - sp, "%s%s:%s",
+				j++ == 0 ? "" : delim,
+				lc,
+				op == 1 ? "on" : "off");
+		if (r < 0 || sp + r >= ep)
+			break;
+		sp += r;
+	}
+}
```

Comment:
Ok - I not understand idea at first. Changed for use  build_cap_string() and lower case result befor output.